src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -74,9 +74,7 @@ export async function fetchSupplementalContextForSrc(
     }
 
     // DO NOT send code chunk with empty content
-    return supplementalContexts.filter(item => {
-        item.content.trim().length !== 0
```

Comment:
I'm surprised that javscript (and eslint) allowed this. Is there an eslint rule we can enable to prevent this?

Reply:
yea lol, agree I was surprised too, not aware of eslint stuff...

Reply:
these may be worth enabling:

- https://eslint.org/docs/latest/rules/no-confusing-arrow
- https://eslint.org/docs/latest/rules/arrow-body-style
- https://eslint.org/docs/latest/rules/no-arrow-condition